project.html

```diff
@@ -282,8 +293,12 @@
 
 <p>
 <span class="header">Grading rubric</span>
+
+Your project will be graded on the following dimensions:
 <ul>
-  <li><b>Task definition</b>: is the task precisely defined and does the formulation make sense?
+  <li><b>Task definition</b>: is the task precisely defined and is the motivation for the task clear?
+    In other words, does the world somehow become a better place if your project were successful?
```

Comment:
Can we change to "...become a better place if your project *(or the vision it hopes to realize)* were successful?

I think it's okay if projects have good intentions, even if they don't directly solve a problem.

Reply:
I think this statement doesn't require success...it requires (success => better place).